sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -171,6 +171,13 @@ static Sampler configureSampler(
               config.getDouble("otel.traces.sampler.arg", DEFAULT_TRACEIDRATIO_SAMPLE_RATIO);
           return Sampler.parentBased(Sampler.traceIdRatioBased(ratio));
         }
+      case "parentbased_jaeger_remote":
+        Sampler jaegerRemote = spiSamplersManager.getByName("jaeger_remote");
+        if (jaegerRemote == null) {
+          throw new ConfigurationException(
```

Comment:
Looks like this line is missing coverage. Should be easy to cover by adding a similar test to the one you have in the vanilla `test` suite (as opposed to `testFullConfig`), which doesn't include the `opentelemetry-sdk-extension-jaeger-remote-sampler` dependency.

Reply:
Fixed, thanks.